plugins/in_prometheus_remote_write/in_prometheus_remote_write_config.c

```diff
@@ -0,0 +1,96 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2015-2024 The Fluent Bit Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <fluent-bit/flb_input_plugin.h>
+#include <fluent-bit/flb_downstream.h>
+
+#include "in_prometheus_remote_write.h"
+#include "in_prometheus_remote_write_conn.h"
+
+/* default HTTP port for prometheus remote write */
+#define PROMETHEUS_REMOTE_WRITE_HTTP_PORT    80
+
+struct flb_in_prometheus_remote_write *in_prometheus_remote_write_config_create(struct flb_input_instance *ins)
```

Comment:
I think the namespace for function name and files is too long "in_prometheus_remote_write" while this model has been used in the first years then we have switched to shorter names for simplicity. I would suggest something like:

**files**

- remote_write.c: `remote_write_config_create()`
- prom_rw.c: `prom_rw_config_create()`

Reply:
I like the latter one. So, I used this abbreviated form.